addons/keybinding/fnc_gui_update.sqf

```diff
@@ -123,12 +121,10 @@ private _tablePosY = 0;
     _edit setVariable [QGVAR(data), [_action, _displayName, _keybinds, _defaultKeybind, _forEachIndex]];
 
     private _assigned = _subcontrol controlsGroupCtrl IDC_KEY_ASSIGNED;
-    _assigned ctrlSetText (_keyNames joinString ", ");
+    _assigned ctrlSetStructuredText parseText (_keyNames joinString ", ");
     _assigned ctrlSetTooltip _tooltip;
-
-    if (_isDuplicated) then {
-        _assigned ctrlSetTextColor [1,0,0,1];
-    };
+    _assigned ctrlSetTooltipColorBox [1,1,1,1];
+    _assigned ctrlSetTooltipColorShade [0,0,0,0.7];
```

Comment:
These already were the values used I believe. Although setting them explicitly may be more robust if future changes happen (e.g. by BI).

Reply:
`RscStructuredText`, unlike `RscText`, has no entries for tooltip defined, and the defaults look different to these. Should probably just move these to config though.